core/src/main/java/org/apache/struts2/views/jsp/SetTag.java

```diff
@@ -59,4 +61,32 @@ public void setScope(String scope) {
     public void setValue(String value) {
         this.value = value;
     }
+
+    public void setTrimBody(boolean trimBody) {
+        this.trimBody = trimBody;
+    }
+
+    public void setStripBodyLineBreaks(boolean stripBodyLineBreaks) {
+        this.stripBodyLineBreaks = stripBodyLineBreaks;
+    }
+
+    @Override
+    protected String getBody() {
+        if (trimBody == true && stripBodyLineBreaks == false) {
+            return super.getBody();
+        } else {
+            String whitespaceModifiedBodyContent = (bodyContent == null ? "" : bodyContent.getString());
+            if (whitespaceModifiedBodyContent == null) {
+                whitespaceModifiedBodyContent = "";
+            } else if (whitespaceModifiedBodyContent.length() > 0) {
+                if (stripBodyLineBreaks) {
+                    whitespaceModifiedBodyContent = whitespaceModifiedBodyContent.replaceAll("[\r\n]+?", "");
+                }
+                if (trimBody == true) {
```

Comment:
same here: `trimBody` to be consistent with L:82

Reply:
Hello @sepe81 .
The `stripBodyLineBreaks` attribute was removed in the latest commit (as per @aleksandr-m 's comments), which allowed for a significant cleanup of the `getBody()` method.  The updated code has the cleaner-style `if` block usage you had asked for.  :)